src/azul/indexer/__init__.py

```diff
@@ -338,11 +338,16 @@ def _aggregate(self, contributions: List[Contribution]) -> List[Aggregate]:
         # Create lookup for transformer by entity type
         transformers = {t.entity_type(): t for t in self.transformers() if isinstance(t, AggregatingTransformer)}
 
+        log.info('Selected %i contribution(s) to be aggregated. Breakdown by entity: %s',
+                 sum(len(contributions) for contributions in contributions_by_entity.values()),
+                 {
+                     f'{entity.entity_type}/{entity.entity_id}': len(contributions)
+                     for entity, contributions in sorted(contributions_by_entity.items())
+                 })
+
         # Aggregate contributions for the same entity
```

Comment:
Since the log statement was move up, I am wondering if this comment is still necessary. I'll leave it to your discretion.

Reply:
Although the log message makes this comment slightly redundant, I think it is worth keeping as it matches with the other comments in the function.